src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -120,6 +105,50 @@ class SubjectHeading extends React.Component {
     this.context.router.push(`${path}/subject_headings/${this.props.subjectHeading.uuid}`)
   }
 
+  updateSortBy(sortByValue) {
```

Comment:
Maybe call this just `updateSort` or `updateSortAttribute`, because `updateSortBy` reads weirdly. Also, `sortType` or `sortAttribute` instead of `sortByValue`? Because the latter sounds more like a method name.

Reply:
renamed